client/src/components/ProfileImage.js

```diff
@@ -2,14 +2,23 @@ import styled from '@emotion/styled';
 import example from '../assets/profile_example.jpg';
 
 const ProfileImage = styled.div`
-  width: 28vh;
-  height: 28vh;
+  width: 185px;
+  height: 185px;
   border-radius: 50%;
   border: solid 1px ${(props) => props.theme.colors.shadow};
   cursor: pointer;
   position: absolute;
   background-image: url(${example});
   background-size: cover;
+
+  @media only screen and (min-device-width: 480px) {
+    width: 200px;
+    height: 200px;
+  }
+  @media only screen and (min-device-width: 768px) {
+    max-width: 230px;
+    max-height: 230px;
+  }
```

Comment:
but then I thought: "aaahahmm I see"

and then I thought, could also work with the `%`, although that would be less "breakpointy"